riscv-asm.md

```diff
@@ -377,6 +377,49 @@ for PIC as seen by objdump:
 			4: R_RISCV_PCREL_LO12_I	.L0
 ```
 
+Load and Store Global
+---------------------
+
+The following pseudo instructions are available to load from and store to
+global objects:
+
+  * `l{b|h|w|d} <rd>, <symbol>`: load byte, half word, word or double word from global
+  * `s{b|h|w|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global
```

Comment:
We don't have support for lq/sq for rv128i yet, but if we did, I would expect them to work the same way.